code/mob/living/life/blood.dm

```diff
@@ -169,10 +173,32 @@
 				if (prob(30))
 					owner.changeStatus("shivering", 6 SECONDS)
 				owner.contract_disease(/datum/ailment/malady/shock, null, null, 1) // if you have no blood you're gunna be in shock
+				APPLY_ATOM_PROPERTY(owner, PROP_MOB_STAMINA_REGEN_BONUS, "hypotension", -5)
+				owner.add_stam_mod_max("hypotension", -50)
+
+			if (1 to 200) // very VERY low (70/50)
```

Comment:
effects here seem to be barely more severe than the 200-300 bracket. Should either make this a bit nastier and/or make the higher bracket more forgiving